internal/fsutil/create.go

```diff
@@ -30,7 +30,8 @@ type Entry struct {
 }
 
 // Create creates a filesystem entry according to the provided options and returns
-// the information about the created entry.
+// the information about the created entry. Create treats a regular file with a
+// non-empty `Link` field as a hard link.
```

Comment:
I think we can improve the documentation by being more detailed about the options. Let's remove the changes here and add them to the `Options.Link` option. What do you think about something like:
```c
	// If Link is set and the symlink flag is set in Mode a symlink is created, else
	// a hardlink is created.
	Link string

```

Reply:
Fixed in [4090b4c](https://github.com/letFunny/chisel/pull/17/commits/4090b4c4c27eda1bd386bb1757572a4529887ad9) and [d3a7921](https://github.com/letFunny/chisel/pull/17/commits/d3a7921df16337e984ea7fb89629d41eedc8da1d).